css/home.css

```diff
@@ -1,4 +1,39 @@
 /* Home page specific styles */
+@media only screen and (min-width: 960px) {
+.footer {
+  padding: 10px;
+  padding-bottom: 4px;
+  background-color: #003c8f;
+  width: 1366px
+  heigth: 270px
+  margin: 0 auto;
+  overflow: hidden;
+  position: relative;
+}
+
+.footer__text {
+  word-break: break-word;
+  text-align: center;
+  font-size: 16px;
+  color:white;
+  font-family: Roboto;
+}
+
+.footer__icons:nth-last-child(1) {
+  margin: 0;
+  display: inline-block;
+  padding:34px;
+}
+
+.footer__hover{
+  color:white;
+}
+
+.footer__icon__element{
```

Comment:
What element do you wanna reference here? Remember that BEM convention is .block__element--modifier, and a second element could have a different approach.

Reply:
ok but is 2-- or only one -?